definition/data/sheets/ComplexTypes.json

```diff
@@ -189,6 +208,7 @@
   {"ID": "Respondent", "ListElementCode": "claimantProvidedRepresentativeOrganisationPhone", "FieldType": "Text", "ElementLabel": "Organisation Phone", "SecurityClassification": "Public"},
   {"ID": "Respondent", "ListElementCode": "claimantProvidedRepresentativeOrganisationDxAddress", "FieldType": "Text", "ElementLabel": "Organisation DxAddress", "SecurityClassification": "Public"},
   {"ID": "Respondent", "ListElementCode": "claimantProvidedRepresentativeOrganisationEmail", "FieldType": "Email", "ElementLabel": "Organisation Email", "SecurityClassification": "Public"},
+  {"ID": "Respondent", "ListElementCode": "directionsQuestionnaire", "FieldType": "DirectionsQuestionnaire", "ElementLabel": "Respondent Directions Questionnaire", "SecurityClassification": "Public"},
```

Comment:
shouldn't we also add the DQ object to the applicant?

Reply:
For applicant it is in `ClaimantResponse`

Reply:
👍 I see